pkg/eks/create.go

```diff
@@ -128,27 +133,44 @@ func CreateStack(opts *CreateStackOptions) (*cloudformation.DescribeStacksOutput
 			return nil, fmt.Errorf("stack did not have output: %v", err)
 		}
 
-		status = *stack.Stacks[0].StackStatus
+		status = string(stack.Stacks[0].StackStatus)
 	}
 
 	if status != createCompleteStatus {
 		reason := "reason unknown"
-		events, err := opts.CloudFormationService.DescribeStackEvents(&cloudformation.DescribeStackEventsInput{
+		events, err := opts.CloudFormationService.DescribeStackEvents(ctx, &cloudformation.DescribeStackEventsInput{
 			StackName: aws.String(opts.StackName),
 		})
 		if err == nil {
 			for _, event := range events.StackEvents {
 				// guard against nil pointer dereference
-				if event.ResourceStatus == nil || event.LogicalResourceId == nil || event.ResourceStatusReason == nil {
+				/*
+					if event.ResourceStatus == nil || event.LogicalResourceId == nil || event.ResourceStatusReason == nil {
```

Comment:
same question about commented statements

Reply:
`event.ResourceStatus` is not a pointer any more
I removed staging comments